client/src/Movies/Movie.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import axios from 'axios';
-import MovieList from './MovieList';
+import MovieCard from './MovieCard';
 
 export default class Movie extends Component {
```

Comment:
I like how you cleaned this part up. Great job on the progress today Shannon! I see you've also attempted the stretches. Keep it up. :) 👍 👍 💯